CatAnalyzer/plugins/TtbarDiLeptonAnalyzer.cc

```diff
@@ -318,9 +310,8 @@ void TtbarDiLeptonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSe
   if      ( b_channel == CH_ELEL ) iEvent.getByToken(trigTokenELEL_, trigHandle);
   else if ( b_channel == CH_MUMU ) iEvent.getByToken(trigTokenMUMU_, trigHandle);
   else if ( b_channel == CH_MUEL ) iEvent.getByToken(trigTokenMUEL_, trigHandle);
-  if ( !trigHandle.isValid() ) return;
+  if ( !trigHandle.isValid() ){ cout <<"error, trigger not vaild "<<endl; }
   b_tri = *trigHandle;
```

Comment:
Actually you don't need to printout this since the line 314 will raise exception automatically and the automatic message contains what which InputTag was asked etc.